core/sql/executor/ExFirstN.cpp

```diff
@@ -228,11 +232,11 @@ short ExFirstNTcb::work()
 		// GET_N request value and firstNRows_ to my child.
 		if ((pentry_down->downState.request != ex_queue::GET_N) ||
 		    (pentry_down->downState.requestValue == firstnTdb().firstNRows()))
-		  centry->downState.requestValue = firstNVal;
+		  centry->downState.requestValue = effectiveFirstN_;
```

Comment:
In the comment above this (line 230 to be precise) should we change "firstNRows_" to "effectiveFirstN_"?